docs/hydra/production.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: production
-title: Prepare for Production
+title: Prepare for production
 ---
 
 This document summarizes some considerations you will find useful when preparing for production.
 
-## Ory Hydra behind an API Gateway
+## Ory Hydra behind an API gateway
 
 Although ORY Hydra implements all Go best practices around running public-facing production http servers, we discourage running
 ORY Hydra facing the public net directly. We strongly recommend running ORY Hydra behind an API gateway or a load balancer. It's
 common to terminate TLS on the edge (gateway / load balancer) and use certificates provided by your infrastructure provider such
 as AWS CA for last mile security.
```

Comment:
```suggestion
Although Ory Hydra implements all Go best practices around running public-facing production HTTP servers, we discourage running
Ory Hydra facing the public net directly. We strongly recommend running Ory Hydra behind an API gateway or a load balancer. It's
common to terminate TLS on the edge (gateway / load balancer) and use certificates provided by your infrastructure provider such
as AWS CA for last mile security.
```

Sorry, spotted that and had to suggest that change :p

Reply:
no need to apologize thats great :D